src/Microsoft.AspNet.Mvc.Core/ControllerActionInvoker.cs

```diff
@@ -30,7 +30,8 @@ public class ControllerActionInvoker : FilterActionInvoker
             [NotNull] IReadOnlyList<IModelValidatorProvider> modelValidatorProviders,
             [NotNull] IReadOnlyList<IValueProviderFactory> valueProviderFactories,
             [NotNull] IScopedInstance<ActionBindingContext> actionBindingContextAccessor,
-            [NotNull] ITempDataDictionary tempData)
+            [NotNull] ITempDataDictionary tempData,
+            int maxAllowedErrorsInModelState)
```

Comment:
`maxModelValidationErrors` might be a bit more clear (consistent with the option name)